host-bin/startkodi

```diff
@@ -17,7 +17,7 @@ Options are directly passed to enter-chroot; run enter-chroot to list them."
 # Forward ports needed to browse smb shares
 MYIP=$(ip route get 1 | awk -F 'src ' '{ split($2,a," ");print a[1];exit}')
 
-if test -n $MYIP; then
+if test $MYIP; then
```

Comment:
`if [ -n "$MYIP" ]; then` should work. It also would be more consistent with the syntax elsewhere used in crouton scripts. Also try to always quote strings.